docs/sources/reference/templating.md

```diff
@@ -114,6 +114,33 @@ String to interpolate: '${servers:percentencode}'
 Interpolation result: 'foo%28%29bar%20BAZ%2Ctest2'
 ```
 
+### Singlequote
+Formats single- and multi-valued variables for use in URL parameters.
+
+```bash
+servers = ['test1', 'test2']
+String to interpolate: '${servers:singlequote}'
+Interpolation result: "'test1','test2'"
+```
+
+### Doublequote
+Formats single- and multi-valued variables for use in URL parameters.
+
+```bash
+servers = ['test1', 'test2']
+String to interpolate: '${servers:doublequote}'
+Interpolation result: '"test1","test2"'
+```
+
+### Sqlstring
+Formats single- and multi-valued variables for use in URL parameters.
+
+```bash
+servers = ["test'1", "test2"]
+String to interpolate: '${servers:sqlstring}'
+Interpolation result: "'test''1','test2'"
```

Comment:
how is this different from singlequote ?

Reply:
Singlequote just quote value with `'`, while sqlstring escape `'` by `''` first, and then quote the value. Sqlstring results is safe in SQL statements. The 2 formats are designed for different scenarios.